python/fastsim/utilities.py

```diff
@@ -394,3 +397,54 @@ def calculate_tire_radius(tire_code: str, units: str = "m"):
     # Return result
     # print(f"Tire radius: {radius} {units}")
     return radius
+
+def show_plots() -> bool:
+    """
+    Returns true if plots should be displayed
+    """
+    return os.environ.get("SHOW_PLOTS", "true").lower() == "true"     
+
+#do I need a different default path?
+def download_demo_files(demo_path: Path=Path("demos")):
```

Comment:
@calbaker any suggestions for how to go about testing/troubleshooting this to see if it works?

Reply:
@robinsteuteville , did you find my ALTRIOS test for this?  Also, since you'll be copying rather than downloading, this is resolved right?

Reply:
@calbaker yes to both those questions